src/main/java/spoon/Metamodel.java

```diff
@@ -0,0 +1,148 @@
+		result.add(factory.Type().get(spoon.reflect.declaration.CtEnumValue.class));
+		result.add(factory.Type().get(spoon.reflect.declaration.CtExecutable.class));
+		result.add(factory.Type().get(spoon.reflect.declaration.CtField.class));
+		result.add(factory.Type().get(spoon.reflect.declaration.CtFormalTypeDeclarer.class));
+		result.add(factory.Type().get(spoon.reflect.declaration.CtInterface.class));
+		result.add(factory.Type().get(spoon.reflect.declaration.CtMethod.class));
+		result.add(factory.Type().get(spoon.reflect.declaration.CtModifiable.class));
+		result.add(factory.Type().get(spoon.reflect.declaration.CtMultiTypedElement.class));
+		result.add(factory.Type().get(spoon.reflect.declaration.CtNamedElement.class));
+		result.add(factory.Type().get(spoon.reflect.declaration.CtPackage.class));
+		result.add(factory.Type().get(spoon.reflect.declaration.CtParameter.class));
+		result.add(factory.Type().get(spoon.reflect.declaration.CtShadowable.class));
+		result.add(factory.Type().get(spoon.reflect.declaration.CtType.class));
+		result.add(factory.Type().get(spoon.reflect.declaration.CtTypeInformation.class));
+		result.add(factory.Type().get(spoon.reflect.declaration.CtTypeMember.class));
+		result.add(factory.Type().get(spoon.reflect.declaration.CtTypeParameter.class));
+		result.add(factory.Type().get(spoon.reflect.declaration.CtTypedElement.class));
+		result.add(factory.Type().get(spoon.reflect.declaration.CtVariable.class));
+		result.add(factory.Type().get(spoon.reflect.declaration.ModifierKind.class));
+		result.add(factory.Type().get(spoon.reflect.declaration.ParentNotInitializedException.class));
+		result.add(factory.Type().get(spoon.reflect.reference.CtActualTypeContainer.class));
+		result.add(factory.Type().get(spoon.reflect.reference.CtArrayTypeReference.class));
+		result.add(factory.Type().get(spoon.reflect.reference.CtCatchVariableReference.class));
+		result.add(factory.Type().get(spoon.reflect.reference.CtExecutableReference.class));
+		result.add(factory.Type().get(spoon.reflect.reference.CtFieldReference.class));
+		result.add(factory.Type().get(spoon.reflect.reference.CtIntersectionTypeReference.class));
+		result.add(factory.Type().get(spoon.reflect.reference.CtLocalVariableReference.class));
+		result.add(factory.Type().get(spoon.reflect.reference.CtPackageReference.class));
+		result.add(factory.Type().get(spoon.reflect.reference.CtParameterReference.class));
+		result.add(factory.Type().get(spoon.reflect.reference.CtReference.class));
+		result.add(factory.Type().get(spoon.reflect.reference.CtTypeParameterReference.class));
+		result.add(factory.Type().get(spoon.reflect.reference.CtTypeReference.class));
+		result.add(factory.Type().get(spoon.reflect.reference.CtUnboundVariableReference.class));
+		result.add(factory.Type().get(spoon.reflect.reference.CtVariableReference.class));
+		result.add(factory.Type().get(spoon.reflect.reference.CtWildcardReference.class));
```

Comment:
Why do you do this by hand instead of retrieving all classes of interesting packages by reflection?